tests/t5_cp.py

```diff
@@ -59,3 +59,62 @@ def tst_cp(self):
 
         finally:
             shutil.rmtree(tempdir)
+
+
+    def test_cp_inode_invalidate(self):
+        if os.getuid() != 0:
+            pytest.skip('test_cp_inode_invalidate requires root, skipping.')
+
+        self.passphrase = None
+        self.mkfs()
+
+        # Create monkeypatched mount.s3ql
+        # Override pyfuse3.invalidate_inode : Drop kernel dentries and inodes cache
+        # just before calling pyfuse3.invalidate_inode
+        patched_mount = """
+import sys
+import os.path
+
+basedir = os.path.abspath(os.path.join(os.path.dirname(sys.argv[0]), '..'))
+sys.path = [os.path.join(basedir, 'src')] + sys.path
+
+import pyfuse3
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
There is no need for that, you can just assign `pyfuse3.invalidate_inode = <...>`

Reply:
Did not know one could just override a method from a lib in python... Done